examples/helia-browser-react-verified-fetch/src/App.tsx

```diff
@@ -1,25 +1,182 @@
-import { useCallback, useState } from 'react'
+// import { verifiedFetch, createVerifiedFetch } from '@helia/verified-fetch'
```

Comment:
when https://github.com/ipfs/helia/pull/423 is merged, we should switch the auto content type button to use the `contentTypeParser` option.